tests/model_hub_tests/transformation_tests/sdpa2pa_ref_diff.py

```diff
@@ -0,0 +1,595 @@
+nodes_to_compare = ("ScaledDotProductAttention", "PagedAttentionExtension", "Parameter", "ReadValue", "Assign")
```

Comment:
why this is of `.py` extension. Is it python file?

Reply:
Yes, this is a Python file that is being included in test_pa_transformation.py later on.